isic/ingest/migrations/0025_auto_20220509_2017.py

```diff
@@ -0,0 +1,74 @@
+# Generated by Django 4.0.3 on 2022-05-09 20:17
```

Comment:
@brianhelba Can you chime in on this? The goal is to rename blob to a randomly generated filename, moving the s3 objects in the process. Right now this only works for minio. What's the recommended way of detecting minio vs s3 (should I even be doing that?) and moving objects?